cmd/sealer/cmd/cluster/delete.go

```diff
@@ -85,8 +83,9 @@ func NewDeleteCmd() *cobra.Command {
 	deleteFlags = &types.Flags{}
 	deleteCmd.Flags().StringVarP(&deleteFlags.Masters, "masters", "m", "", "reduce Count or IPList to masters")
 	deleteCmd.Flags().StringVarP(&deleteFlags.Nodes, "nodes", "n", "", "reduce Count or IPList to nodes")
+	deleteCmd.Flags().StringVarP(&deleteClusterFile, "Clusterfile", "f", "", "delete a kubernetes cluster with Clusterfile")
```

Comment:
default is common.GetDefaultClusterfile()

Reply:
> 

"sealer delete -a" deletes files under ".sealer/Clusterfile" by default

If you specify the default delete path, "sealer delete -f" loses the meaning of the specified file